changelog.json

```
@@ -1,5 +1,21 @@
 {
   "releases": {
+    "1.7.0-beta0": [
+      "[New] Repository list now displays recent repositories at top - #7132",
+      "[Fixed] Error when selecting diff text while diff is updating - #7131",
+      "[Fixed] Crash when unable to create log file on disk - #7096",
+      "[Fixed] Race condition with remote lookup could cause push to go to incorrect remote - #6986",
+      "[Fixed] Fix mistaken classification of all crashes as being related to launch - #7126",
+      "[Fixed] Improve user lookup to handle incomplete API response - #6945",
```

I'm less certain about this but I do think we can be less technical in these and tell people how they're actually impacted.
```suggestion
      "[Fixed] Incomplete information from API caused menu items to be greyed out - #6945",
```

---

i think this should actually be removed as the parent issue is captured in the other "menu items grayed out" release note
```suggestion
```